src/BusyImage.tsx

```diff
@@ -75,8 +75,16 @@ const BusyImage = (props: BusyImageProps) => {
     const rect = ev.currentTarget.getBoundingClientRect();
 
     // set the label position relative to the BusyImg
-    setHoveredLabelX(ev.clientX - rect.x + ev.currentTarget.scrollLeft + 10);
-    setHoveredLabelY(ev.clientY - rect.y + 10);
+    if (ev.clientX + ev.currentTarget.scrollLeft > ev.currentTarget.scrollWidth/2){
```

Comment:
Not a big deal, but I think we could avoid the popping by using a min function here. 

For example:
```
const MAX_LABEL_WIDTH = 200; // in pixels
const targetX = ev.clientX - rect.x + ev.currentTarget.scrollLeft + 10; 
setHoveredLabelX(min(targetX, imgWidth - MAX_LABEL_WIDTH));
...
```

That way as targetX gets closer to that boundary it uses the fixed edge.